src/borough/borough.service.ts

```diff
@@ -14,4 +15,18 @@ export class BoroughService {
       boroughs,
     };
   }
+
+  async findCommunityDistrictsByBoroughId(id: string) {
+    const boroughCheck = await this.boroughRepository.checkBoroughById(id);
+    if (boroughCheck === undefined) throw new ResourceNotFoundException();
+
+    const communityDistricts =
+      await this.boroughRepository.findCommunityDistrictsByBoroughId(id);
+
+    if (communityDistricts === undefined) throw new ResourceNotFoundException();
```

Comment:
if `findCommunityDistrictsByBoroughId` is defined in the repository as only being able to return `Promise<FindCommunityDistrictsByBoroughIdRepo>` and not `Promise<FindCommunityDistrictsByBoroughIdRepo | undefined>` can we still check for undefined here?

Reply:
> can we still check for undefined here

If you mean "we don't need to check for undefined", then I agree on removing this undefined check

Reply:
Oops, sorry! This was left over after I refactored the function DH initially wrote. It definitely should have been removed.